service.go

```diff
@@ -108,9 +108,9 @@ func (s *service) Init(opts ...Option) {
 
 		// If the store has no namespace set, fallback to the
 		// services name
-		if len(store.DefaultStore.Options().Namespace) == 0 {
+		if len(store.DefaultStore.Options().Database) == 0 {
 			name := s.opts.Cmd.App().Name
-			store.DefaultStore.Init(store.Namespace(name))
+			store.DefaultStore.Init(store.Database(name))
```

Comment:
This should be set to Table name since we're having table per service rather than database per service